etest_test/build_test.py

```diff
@@ -1,12 +1,236 @@
 """Tests for etest.build."""
 
+import logging
+import textwrap
+
 import click.testing
+import docker
+import pytest
 
 import etest.build as sut
+from etest import profile
+
+logger = logging.getLogger(__name__)
 
 
 def test_help() -> None:
     """Ensure etest-build --help exits successfully."""
     runner = click.testing.CliRunner()
     result = runner.invoke(sut.main, ["--help"])
+
+    assert result.exit_code == 0
+
+
+def test_nobuild() -> None:
+    """Ensure etest-build outputs 4 info lines when not building."""
```

Comment:
This is a bad thing to check as it couples the test to the particulars of the implementation.  How can we verify that the command ran as expected rather than just emitted the logging we check for?